src/api/layers/ExternalWMSLayer.class.js

```diff
@@ -68,6 +68,7 @@ export default class ExternalWMSLayer extends ExternalLayer {
             hasTooltip = false,
             getFeatureInfoCapability = null,
         } = externalWmsData
+
```

Comment:
left over of some changes? doesn't really matter, but if you have to push once again maybe clean that up :wink: